bin/cron/deploy_to_test

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env ruby
+
+require 'cdo/chat_client'
+require 'cdo/github'
+require 'cdo/only_one'
+
+# @return [Boolean] Whether  the topic of Slack#developers includes "DTT: yes".
+def dtt_yes?
+  current_topic = Slack.get_topic('developers')
+  current_topic.include? 'DTT: yes'
+end
+
+# Updates the DTT portion of topic of Slack#developers to indicate robo-DTT.
+def update_developers_topic_robo_dtt
+  current_topic = Slack.get_topic('developers')
+  raise unless current_topic.include? 'DTT: yes'
+  new_topic = current_topic.gsub('DTT: yes', 'DTT: no (robo-DTT in progress)')
```

Comment:
Nit for future updates: We reuse a bunch of these strings two or three times, and it'd be nice to extract them to constants to avoid typos, e.g.
```ruby
TOPIC_DTT_YES = 'DTT: yes'.freeze
TOPIC_DTT_IN_PROGRESS = 'DTT: no (robo-DTT in progress)'.freeze
TOPIC_DTT_FAILED = 'DTT: robo-DTT failed'.freeze
```